Source/WTF/wtf/text/CStringView.h

```diff
@@ -116,6 +114,9 @@ inline bool operator==(ASCIILiteral a, CStringView b)
 // CStringView is null terminated
 inline const char* safePrintfType(const CStringView& string) { return string.utf8(); }
 
+WTF_EXPORT_PRIVATE String toString(CStringView);
```

Comment:
The constructor for `String` that takes a `std::span<const char8_t>` is now implemented, so we don't need this.